src/Framework/test/SharedFxTests.cs

```diff
@@ -137,5 +138,57 @@ public void ItContainsVersionFile()
             Assert.Equal(TestData.GetRepositoryCommit(), lines[0]);
             Assert.Equal(TestData.GetSharedFxVersion(), lines[1]);
         }
+
+        [Fact]
+        public void PlatformManifestListsAllFiles()
```

Comment:
Wasn't aware we ran tests for SharedFx, this test is ported from TargetingPackTests and covers the PlatformManifest I added to the runtime pack.

Reply:
FYI @jkotalik  

Yes, we have some limited testing for the manifests that we generate. Errors in the manifests can cause big issues, and we've had regressions in this area before, so automated testing is super helpful.

Reply:
Is this tangential to the rollforward change?

Reply:
This test is actually following up to my previous change https://github.com/aspnet/AspNetCore/commit/7a8487e2bdbe72c0aa3146ca0b82842da99eec0f. But I'm including it here since I don't want to make a separate PR.

Reply:
Cool. Should we be responsible for adding a test to make sure roll forward works?

Reply:
Seems like there are some clarification needed in this area. I'll hold off on adding the tests for now. As for the roll-forward behaviour tests, I think that belongs in core-host that actually interprets this setting. It is sufficient to test that the setting exists in our tests.